lib/Engineer.js

```diff
@@ -0,0 +1,20 @@
+const Employee = require("./Employee.js")
+
+class Engineer extends Employee {
+    constructor(name, email, param) {
+        super(name, email)
+        this.role = "Engineer";
+        this.github = param.github;
+        this.extraInfoProper = "Github";
+        this.icon = '<i class="bi bi-gear"></i>';
+    }
+    getExtraInfo() {
+        var github = this.getGithub();
+        return `<a href="${github}">${github}</a>`
```

Comment:
ideally, try to only return a github username here, in place of the entire html element as this will help to make your code more modular.